mi/core/instrument/playback.py

```diff
@@ -378,9 +378,9 @@ def main():
         reader = None
 
     wrapper = PlaybackWrapper(module, refdes, event_url, particle_url, reader, allowed, files, max_events)
-    if zplsc_reader:
+    try:
```

Comment:
I don't believe this is the correct solution.  zplsc_reader simply needs to be initialized to false prior to the if statement on line 368.

With the try statement, it is being used as a conditional statement so when the zplsc_reader isn't initialized and the exception is thrown, playback is executed rather than zpsc_playback.

I believe the zplsc_reader flag is ok, it just needs to be initialized.